### PR TITLE
Fix #706, Add location to push CTF scripts and plugins

### LINKED ISSUE #706: Create directory structure for pushing CTF scripts
**Is your feature request related to a problem? Please describe.**
Need to create a location to push CTF scripts and plugins to.

**Requester Info**
Dan Knutsen
NASA Goddard

### PR DESCRIPTION
**Pull request by dmknutsen**
**Describe the contribution**
Fix #706 
Adds locations to push CTF scripts and plugins

**Testing performed**
Steps taken to test the contribution:
1. Ensured that the editor launches nominally
2. Built and executed 3 test scripts

**System(s) tested on**
Oracle VM VirtualBox
OS: ubuntu-19.10
Versions: cFE 6.7.13.0, OSAL 5.0.8.0, PSP 1.4.4.0

**Contributor Info - All information REQUIRED for consideration of pull request**
Dan Knutsen
NASA/Goddard

### CODE REVIEW

File: ctf/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Getting Started
+
+To get started, clone the CTF repository using:
+
+git clone git@aetd-git.gsfc.nasa.gov:gateway_cert/cfs_gw.git
+cd cfs_gw
+git submodule update --init
+
+Note that the CTF repository must be cloned at the same directory level as the cFS repository.
```
Comment (skliper) [inline ctf/README.md:14]: We shouldn't include the internal steps for the open source instructions.  Open source steps would be to clone cFS and clone CTF from public repos.

Comment (skliper) [reply ctf/README.md:14]: And CTF hasn't been open source released yet... but leave as a TBD?

**Comment by astrogeco** [comment]
CCB 20200513 - Move to a stand alone repository

**Comment by astrogeco** [comment]
@dmknutsen do you need me to request the new repository under the NASA org or do you want to do it yourself?

**Comment by dmknutsen** [comment]
@astrogeco Jake set one up last week.  It is located here:
https://aetd-git.gsfc.nasa.gov/gateway_cert/cfe_bvt

**Comment by astrogeco** [comment]
> @astrogeco Jake set one up last week. It is located here:
> https://aetd-git.gsfc.nasa.gov/gateway_cert/cfe_bvt

@skliper I thought we agreed to make it a github repo?

**Comment by skliper** [comment]
@astrogeco went local for simplicity for now, we can move to github later.

**Comment by skliper** [comment]
Closing as invalid since it won't go in this repo

**Pull request closed without merging**